### ISSUE TITLE
Live Coders leadership bringing people to the team outside outside application process

### ISSUE THREAD

**Issue by lannonbr** [livecoders/Home#350]
For over a year now it seems, there has been over now 50 applicants who have applied to the team. None of them have seemed to been accepted / declined, but further. There's been people being added to the team who never applied in this repo. (Context, I am a former member of the team and set up a good portion of the website. There's scripts that run for the site that generates markdown files for team members when a new member joins the team):

- https://github.com/livecoders/website/commit/e79cdc0676ce575322d0320260027e32bcb16d34#diff-896910827fa03e458d8e9aa5e80bed0e1cd08e63009436b571eb6a840317cd31
- https://github.com/livecoders/website/commit/078fcede46258fe863c0585476fbe7b8c3831233#diff-896910827fa03e458d8e9aa5e80bed0e1cd08e63009436b571eb6a840317cd31
- https://github.com/livecoders/website/commit/0d59ae8d06073c77206cd6a59fd3b331293b6189#diff-896910827fa03e458d8e9aa5e80bed0e1cd08e63009436b571eb6a840317cd31

There is a system for joining the team and even if the applications were paused after a good portion of the team decided to leave last year after poor decisions by leadership, inviting people behind the scenes outside the application process downplays people trying to join the team in official ways.

**Comment by csharpfritz** [comment]
That’s right.. folks who were in the mentor ship program were added to the team. 

For someone who decided to leave the organization, we find your continued interest in team management flattering

**Comment by lannonbr** [comment]
I'm sorry that I didn't have the context they were partially part of the team already, but still it feels like things like the application process here and the maintainence of things like the team website is a bit lacking.
